### DISCUSSION TITLE
How do I add HTML file into next.js app

### DISCUSSION THREAD

**Discussion by VelrinBlack** [vercel/vercel#5338] [Help] [answered]
Hello,

I've deployed the next.js app to vercel. It uses SSR for most routes, so I couldn't upload static files.
Now I have to place an HTML file on the server. It is file from Google Search Console, but I think it shouldn't matter. The important thing might be, that there's no regular HTML structure in it. There's only single line of text.

How can I do that without hosting static files?

Thanks in advance

**Comment by pueyo5** [comment, marked as answer]
For this purpose you can use the Next.js public folder

**Reply by VelrinBlack** [reply]
Thanks man
